doc/sources/installation/installation-windows.rst

```diff
@@ -230,6 +239,36 @@ kivy with git rather than a wheel there are some additional steps:
      export USE_GSTREAMER=1
 
    These variables must be set everytime you recompile kivy.
+   
+   **MSVC:**
```

Comment:
This whole msvc section should not be needed if you installed visual studio, which we officially support. If using the build tools, we don't officially support it so it should be added to the wiki instead, which is why I removed those instruction yesterday and moved it to the linked wiki. It's best if we only officially support the visual studio as that should work out of the box without anything else required. Python will automatically find it.